app/web/src/newhotness/AddComponentModal.vue

```diff
@@ -0,0 +1,359 @@
+                    themeClasses(
+                      'bg-shade-0 hover:outline-action-500',
+                      'bg-neutral-800 hover:outline-action-300',
+                    ),
+                  )
+                "
+                :color="asset.color"
+              >
+                <template #label>
+                  <!-- TODO(Wendy) - style this text based on the fuzzy search! -->
+                  {{ asset.name }}
+                </template>
+              </TreeNode>
+            </TreeNode>
+          </div>
+        </div>
+      </div>
+      <!-- Right side - documentation and attributes -->
+      <div
+        v-if="selectedAsset"
+        :class="
+          clsx(
+            'information flex flex-col gap-xs [&>*]:border [&>*]:p-xs [&>*]:grow',
+            themeClasses(
+              '[&>*]:border-neutral-400',
+              '[&>*]:border-neutral-600',
+            ),
+          )
+        "
+      >
+        <div>Documentation goes here</div>
+        <div>Attributes go here</div>
+      </div>
+    </div>
+  </Modal>
+</template>
+
+<script setup lang="ts">
+// import { Fzf } from "fzf"; // TODO(Wendy) - import this and use it!
+import {
+  BRAND_COLOR_FILTER_HEX_CODES,
+  HorizontalScrollArea,
+  Icon,
+  IconButton,
+  IconNames,
+  Modal,
+  SiSearch,
+  themeClasses,
+  TreeNode,
+} from "@si/vue-lib/design-system";
+import { computed, inject, ref } from "vue";
+import clsx from "clsx";
+import { useQuery } from "@tanstack/vue-query";
+import TextPill from "@/components/TextPill.vue";
+import { BifrostSchemaVariantCategories } from "@/workers/types/dbinterface";
+import { bifrost, makeArgs, makeKey } from "@/store/realtime/heimdall";
+import ComponentFilterTile from "./ComponentFilterTile.vue";
+import { Context } from "./types";
+
+const ctx: Context | undefined = inject("CONTEXT");
+const onHead = computed(() => ctx?.onHead ?? false); // TODO(Wendy) - doesn't seem to be working? Not sure why.
+const bannerClosed = ref(false);
+const selectedAsset = ref<UIAsset | undefined>(undefined);
+
+export type AssetFilter = {
+  name: string;
+  icon: IconNames;
+  count: number;
+  color?: string;
+};
+
+type UICategory = {
+  name: string;
+  icon?: IconNames;
+  color: string;
+  assets: UIAsset[];
+};
+
+type UIAsset = {
+  id: string;
+  name: string;
+  color: string;
+};
+
+const queryKey = makeKey("SchemaVariantCategories");
+const schemaVariantCategoriesOverBifrost =
+  useQuery<BifrostSchemaVariantCategories | null>({
+    queryKey,
+    queryFn: async () =>
+      await bifrost<BifrostSchemaVariantCategories>(
+        makeArgs("SchemaVariantCategories"),
+      ),
+  });
+
+const categories = computed(() => {
+  const rawCategoryData =
+    schemaVariantCategoriesOverBifrost.data.value?.categories ?? [];
+
+  return rawCategoryData.map((rawCategory) => {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const firstSV = rawCategory.schemaVariants[0]!;
```

Comment:
Note for the future: I think this MV should just have the default one available immediately rather than us always grabbing the first
cc @jobelenus